aten/src/ATen/cuda/CUDASparseDescriptors.cpp

```diff
@@ -12,8 +12,6 @@ cusparseStatus_t destroyConstDnMat(const cusparseDnMatDescr* dnMatDescr) {
   return cusparseDestroyDnMat(const_cast<cusparseDnMatDescr*>(dnMatDescr));
 }
 
-#if AT_USE_CUSPARSE_GENERIC_API() || AT_USE_HIPSPARSE_GENERIC_API()
```

Comment:
might as well just remove these definitions then as they look even older?

Reply:
I just encountered HIP errors in removing them. So I would like to fix the current build errors before considering that.